### PR TITLE
Added acronym problem

### PR DESCRIPTION
**Pull request by monkbroc**
I had an idea for a problem: shorten a string to its acronym. Like Portable Network Graphics :arrow_right: PNG.

I wrote tests and an example solution in Ruby. I don't know where a README should go, but I'm submitting this so you can review the idea.

### CODE REVIEW

**Comment by kotp** [comment]
The readme goes into https://github.com/exercism/x-common as per the README file.

**Comment by monkbroc** [comment]
Ok. Thanks. Should I make a PR to that repository or wait until this new
problem is discussed and ok'ed before doing that?

**Comment by kotp** [comment]
Go ahead and create the README with a pull request, so @kytrinyx can see the whole picture.  Then it is a simple accept or deny action if it is that simple, and any issues can be addressed at the proper location, perhaps with references to the other in the comments.

**Comment by monkbroc** [comment]
OK. I added the metadata to [the x-common repository in this pull request.](https://github.com/exercism/x-common/pull/61).

I also added the new problem to the Ruby track to get the Travis build to pass.

**Comment by kytrinyx** [comment]
Neat!

In terms of the API, I think I'd prefer to make it a class or module function. Then people can decide whether or not to delegate to an instance or to leave it as a purely functional thing.

Also, how about `abbreviate` rather than `short`?

**Comment by kytrinyx** [comment]
Oh, one more thing. Let's move it even earlier in the progression, perhaps right after raindrops?

File: config.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
     "scrabble-score",
     "nucleotide-count",
     "hexadecimal",
+	"acronym",
```
Comment (kytrinyx) [inline config.json]: Would you fix the indentation here?

**Comment by monkbroc** [comment]
I renamed the method to `abbreviate` and made it a class method.

When you say "put it earlier in the progression" I assume it means the problems are supposed to go in increased difficulty? It's not something that comes across to me at all. It felt more like one exercise after the other, all of them fairly straightforward. Is it something that is important to the experience of Exercism in your opinion?

**Comment by kytrinyx** [comment]
> I assume it means the problems are supposed to go in increased difficulty

I can only say "ish". For an experienced programmer they're all trivial, but for beginners there are definitely concepts that get harder further out. It's very hand-wavey, and I don't know that it's core to the experience.

Merging. Thanks!

**Comment by monkbroc** [comment]
Yeah, my feeling while doing the Ruby exercises is that the best ones are the ones that give more opportunities to comment on style while not being tedious. Difficulty doesn't seem to play a big role.

**Comment by kytrinyx** [comment]
Agreed. Some problems have a lot of different opportunities, others are like "write this one liner. Okay? Okay!"

I'd actually be interested in figuring out which ones could be retired.

**Comment by monkbroc** [comment]
What about a simple +1/-1 vote button for each exercise? "This is a great exercise", "Not so good"

This way the data will show which ones are most appreciated and which ones are most disliked.

**Comment by kytrinyx** [comment]
Yeah, I think that could work. We could have it both in the exercise page and on the dashboard (for exercises that you haven't voted on). Would you mind opening a new issue for this?

**Comment by kytrinyx** [comment]
Actually, maybe this is part of this discussion: https://github.com/exercism/exercism.io/issues/1081

**Comment by monkbroc** [comment]
@kytrinyx This problem is too early in the progression. The combination of test cases is causing headaches to people not too familiar with regular expressions. [One person said he worked on it for 90 minutes and couldn't get all test cases to pass.](http://exercism.io/submissions/d618971d10984a1fb59c5c3596047ecd)

I think it should be moved back later or the problem made easier. What's your thought?

**Comment by kytrinyx** [comment]
Wow, yeah, 'm happy to move it to a later spot.

**Pull request merged**